packages/devtools_app/lib/src/ui/service_extension_widgets.dart

```diff
@@ -889,6 +889,9 @@ class ServiceExtensionTooltip extends StatelessWidget {
         borderRadius: BorderRadius.circular(defaultBorderRadius),
       ),
       textStyle: DefaultTextStyle.of(context).style,
```

Comment:
use the same text color as the other tooltips on the Inspector page: https://github.com/flutter/devtools/blob/374f37db70ab18acf1055807c4f59d14e85d24ce/packages/devtools_app/lib/src/ui/service_extension_widgets.dart#L919. 

You can pass in a TextStyle just like the text widget I linked passes in a TextStyle with this color:
`TextStyle(color: Theme.of(context).colorScheme.toggleButtonsTitle)`